stack.go

```diff
@@ -148,32 +155,47 @@ func (c Call) Format(s fmt.State, verb rune) {
 // PC returns the program counter for this call frame; multiple frames may
 // have the same PC value.
 func (c Call) PC() uintptr {
-	return c.pc
+	return c.pcs[1]
+}
+
+func (c Call) frame() (*runtime.Frame) {
+	frames := runtime.CallersFrames(c.pcs[:])
+
+	frame, _ := frames.Next()
+	next, _ := frames.Next()
+	if frame.Function == "runtime.sigpanic" {
+		frame, _ = runtime.CallersFrames([]uintptr{next.PC - 1}).Next()
```

Comment:
This should be unnecessary. `frames.Next` handles the checks for `sigpanic` frames and makes the necessary adjustment internally.

See https://github.com/go-stack/stack/pull/13#discussion_r122351829 for how I suggested implementing this method before.